### PR TITLE
Fixed text for the PID profile reset button.

### LINKED ISSUE #2598: "Reset all profiles" doesn't work on Rateprofile settings
To reproduce:

- come to pid tuning->rateprofile settings
- change rates settings
- save
- hit "Reset all profiles" button

Expected:
- rates are reseted to default

Actual:
- rates still changed

reproduces in 10.7 and master branches

### PR DESCRIPTION
**Pull request by mikeller**
Fixes #2598.

It's a bit wordy, but concise, making it clear that only the current PID profile is affected: 

image

### CODE REVIEW

File: src/js/tabs/pid_tuning.js [hunk 1]
```diff
@@ -1364,15 +1364,15 @@ TABS.pid_tuning.initialize = function (callback) {
...
+        $('#resetPidProfile').on('click', function(){
             self.updating = true;
             self.sliderRetainConfiguration = true;
 
             MSP.promise(MSPCodes.MSP_SET_RESET_CURR_PID).then(function () {
                 self.refresh(function () {
                     self.updating = false;
 
-                    GUI.log(i18n.getMessage('pidTuningProfileReset'));
+                    GUI.log(i18n.getMessage('pidTuningPidProfileReset'));
```
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:1375]: Is this message valid for previous firmware? Otherwise we should maintain both and check with semver.

File: src/css/tabs/pid_tuning.css [hunk 2]
```diff
@@ -488,7 +488,7 @@
 }
 
 .tab-pid_tuning .resetbt {
-    width: 140px;
+    width: 200px;
```
Comment (haslinghuis) [inline src/css/tabs/pid_tuning.css:491]: Maybe `fit-content` would work also (if we have different messages, otherwise ignore)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS
